### PR TITLE
is_admin to is_owner and group specs

### PR DESCRIPTION
**Pull request by RobertAKARobin**

### CODE REVIEW

**Comment by RobertAKARobin** [comment]
Forgot to `git add db/schema.rb`

**Comment by RobertAKARobin** [comment]
Note: The vast majority of these changes are simply caused by renaming the `is_admin` column to `is_owner`, which I think is more semantic. The terminology I'd like to use is:

-*owner*: A member of a group where `is_owner` is true
-*nonowner*: A member of a group where `is_owner` is false
-*admin*: An owner of a group, or of any of its ancestor groups
-*nonadmin*: A nonowner of a group, or any member of any of its descendant groups

File: app/models/group.rb [hunk 1]
```diff
@@ -85,7 +94,7 @@ def has_member? user
   end
 
   def has_admin? user
-    self.ancestors.collect(&:memberships).flatten.select(&:is_admin).collect(&:user).include?(user)
+    self.admins.include?(user)
```
Comment (mattscilipoti) [inline app/models/group.rb]: :heart_eyes:

**Comment by RobertAKARobin** [comment]
Needed to update tests to match the seeds created by Matt's commit

**Comment by mattscilipoti** [comment]
:+1:

**Comment by andrewsunglaekim** [comment]
:+1:

**Comment by RobertAKARobin** [comment]
Yizzzz

**Pull request merged**